js/index.js

```diff
@@ -208,22 +204,20 @@
         };
 
         if (!BR.conf.transit) {
-            map.addControl(new BR.Control({
-                 heading: '',
-                 divId: 'header'
-            }));
+            console.log("BR.Control: fix me");
+            // map.addControl(new BR.Control({
+                 // heading: '',
+                //  divId: 'header'
+            // }));
```

Comment:
One thing that has to be fixed. Not sure how though since I don't know what BR.Control is supposed to look like.